src/zapv2/__init__.py

```diff
@@ -111,9 +113,11 @@ def __init__(self, proxies=None, apikey=None, validate_status_code=False):
         self.httpsessions = httpSessions(self)
         self.localProxies = localProxies(self)
         self.network = network(self)
+        self.oast = oast(self)
         self.openapi = openapi(self)
         self.params = params(self)
         self.pnh = pnh(self)
+        self.postman = postman(self)
```

Comment:
Better not include this one, it's not functional.

Reply:
Shall I put in a quick PR that removes that?

Reply:
I can update my PR. I'll need to rebase on main anyway .. once the build works in that branch...

Reply:
I meant to also remove it at the source, zap-extensions.

Reply:
That or we could actually implement the API 😁